R/plot_missing.r

```diff
@@ -26,6 +26,10 @@
 
 plot_missing <- function(data,
                          group = list("Good" = 0.05, "OK" = 0.4, "Bad" = 0.8, "Remove" = 1),
+                         fill_color = c("Good"   = "#1B9E77",
```

Comment:
This will work for the default case, but will break if number of groups is not 4.

Reply:
Good point! I think I only need to add error messages when groups' names are different from fill_color's names to make them aware of their errors, most people will use the default groups.